### DISCUSSION TITLE
DISC_DIP & DISC_DIR

### DISCUSSION THREAD

**Discussion by richard-sisson** [australiangeomechanics/agsformat#61] [Q&A] [answered]
Hi,

The descriptions for 'DISC_DIP' (dip) & 'DISC_DIR' (dip direction) indicate these are 'apparent' values, differing from other AGS schema that provide no additional guidance. 

For me, the term 'Dip & Dip Direction' infers that these are 'true' (corrected) values.

I appreciate the need to store and report uncorrected data of 'apparent' (outcrop), 'relative' (non-oriented core) and 'alpha/beta' (oriented core), but I feel the new headings defined in AGS AU should be for storing these uncorrected values while preserving DISC_DIP and DISC_DIR for 'true' values.  

We can work with your setup, I'm just interested to understand the reasoning behind your decision.

Thanks
Richard

**Comment by SThorin** [comment, marked as answer]
Thank you for your relevant comment. Historically, the AGS UK fields have been used in Australia for apparent dips in almost all cases. True dips are rarely available and when provided, are often unreliable (unless downhole televiewer data is available and a detailed correlation exercise has been carried out. This correlation is often done at interpretive stage by other users. We want to avoid mixing fields). As you mention, AGS UK has not specified true or apparent. This is what led to the decision of keeping the AGS UK fields as apparent dips.

**Reply by richard-sisson** [reply]
Thanks for the explanation.
Thanks
Richard

**Reply by ijaukovic** [reply]
Hi Richard and Sven,
Thanks for the info in this post! I discovered this while working through some queries on apparent and true dips for non-oriented and oriented core.
Does AGS AU have guidance on how to record the plane of reference for the apparent dip and dip direction?
i.e. relative to core axis / reference line (ori), or, relative to a plane perpendicular to the core axis (non-ori)?
I have previously recorded this as a Depth Related Remark (DREM) at the top of the cored interval but this is then not associated with the DISC Group nor does it account for changes downhole (i.e. initially measured relative to core-horizontal, then oriented from X depth) and can be lost over subsequent log sheets.
Thanks in advance, 
Isabella

**Reply by SolidRockProgrammer** [reply]
Hi Isabella
Can you state your request with reference to the requirements of AS 1726?  Perhaps list the heading names, description, and unit you propose. If any...

the Australia notes for Guidance states, on Groups tab
• Dips should be provided as **apparent dips in DISC_DIP, relative to a plane perpendicular to the core axis,** even for inclined boreholes. When measured (e.g. goniometer, downhole imaging), true dips and maximum dip directions should be provided separately in the true dip headings DISC_DIPT and DISC_DIRT.
• DISC_DIP and DISC_DIR data type changed to X but care should be taken to ensure a consistent convention is used throughout the AGS file. Where possible one value should be reported but if a range is required the presence of excess spaces and inconsistent separators is discouraged e.g. “150-160” not “150 – 160” or “150 to 160”

regards

Phil
